medusa/tv/series.py

```diff
@@ -2131,19 +2155,31 @@ def to_json(self, detailed=True):
             data['config']['release']['whitelist'] = bw_list.whitelist
 
         if detailed:
-            episodes = self.get_all_episodes()
             data['size'] = self.size
-            data['seasons'] = [list(v) for _, v in
-                               groupby([ep.to_json() for ep in episodes], lambda item: item['season'])]
+            data['showQueueStatus'] = self.show_queue_status
+            data['xemNumbering'] = [{'source': {'season': src[0], 'episode': src[1]},
+                                     'destination': {'season': dest[0], 'episode': dest[1]}}
+                                    for src, dest in viewitems(self.xem_numbering)]
+            data['sceneAbsoluteNumbering'] = self.scene_absolute_numbering
+            data['allSceneExceptions'] = self.all_scene_exceptions
+            if self.is_scene:
+                data['xemAbsoluteNumbering'] = self.xem_absolute_numbering
+                data['sceneNumbering'] = [{'source': {'season': src[0], 'episode': src[1]},
+                                           'destination': {'season': dest[0], 'episode': dest[1]}}
+                                          for src, dest in viewitems(self.scene_numbering)]
+            else:
+                data['xemAbsoluteNumbering'], data['sceneNumbering'] = [], []
```

Comment:
I think splitting this over two lines in this case is more readable.
```suggestion
                data['xemAbsoluteNumbering'] = []
                data['sceneNumbering'] = []
```